.gitignore

```diff
@@ -0,0 +1,16 @@
+cursor.png
```

Comment:
As Mike was saying, only add files here that you don't want to add to your repo. If you need these image files, don't put them in your .gitignore file.